scripts/screener/screener.config.js

```diff
@@ -2,6 +2,30 @@
 
 require('@uifabric/build/babel/register');
 
+const cp = require('child_process');
```

Comment:
This comment applies to more than just this one file, but I think it would be helpful to have a better way to denote which files only apply for the fluent build. Especially if we end up in this state for a while, it will confuse anyone coming in and trying to understand the build tools if there are two completely separate build systems combined into the same folder. (This is kind of a similar problem to how the scripts specific to docs make the overall fluent build look a lot more complicated than it is.)

Reply:
I'm taking a simple approach to not block us from repo merging. We need to quickly move files around once everyone's on this one repo.